src/YoutubeAtomOverlay.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { pillarPalette } from './lib/pillarPalette';
+
+import {
+    focusHalo,
+    space,
+    textSans,
+    neutral,
+} from '@guardian/source-foundations';
+import { SvgPlay } from '@guardian/source-react-components';
+
+import { formatTime } from './lib/formatTime';
+import { Picture } from './Picture';
+import { ImageSource, RoleType } from './types';
+import type { ArticleTheme } from '@guardian/libs';
+
+type Props = {
+    overrideImage?: ImageSource[];
+    posterImage?: ImageSource[];
+    height: number;
+    width: number;
+    alt: string;
+    role: RoleType;
+    duration?: number; // in seconds
+    pillar: ArticleTheme;
+    overlayClicked: boolean;
+    setOverlayClicked: (flag: boolean) => void;
```

Comment:
Good idea thanks - it makes the overlay less semantically coupled.

I've lifted `overlayClicked` up and created an `onClick` callback.

The only drawback is that because `YoutubeAtom` now decides whether to render `YoutubeAtomOverlay` (and not itself) we lose the fade-out transition that was in `YoutubeAtomOverlay`.

imo It was barely visible and if anything a little distracting as the player has its own fade-in transition when it loads.

Something to keep an eye on though.